packages/components/src/mobile/audio-player/test/audio-url-parser.native.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Internal dependencies
+ */
+import { parseAudioUrl } from '../audio-url-parser.native';
+
+const supportedAudioUrlsWithExtensions = [
+	'https://www.mp3.com/file.mp3?key1=value1&key2=value2#anchorforsomereason',
+	'https://www.mp3.com/file.mp3?key1=value1&key2=value2',
+	'https://www.mp3.com/file.mp3',
```

Comment:
It would be nice if we could also check the case where the URL contains a folder:
```
https://www.mp3.com/folder/file.mp3
```

Reply:
Added here 05a8335